README.md

```diff
@@ -88,7 +92,8 @@ Verloop expects the response to be in the following format.
           "variables": {
             "operator": "Vodafone"
           }
-        }
+        },
+        "triggers":["Zoozoo","Pug"]
       }
```

Comment:
Are these triggers case sensitive?

Reply:
I just tried this and they are case insensitive, thanks!

Reply:
Hey yeah they are case insensitive. The behaviour is similar to how it's in UI.